app/controllers/prison/cancellations_controller.rb

```diff
@@ -0,0 +1,41 @@
+class Prison::CancellationsController < ApplicationController
+  include StaffResponseContext
```

Comment:
Seems a bit overkill to include the `StaffResponseContext` since it is very specific to logic to `requested` visits. You are only using the `memoised_visit` method right?

Reply:
I'm using the `#memoized_visit` (relies on the `#scoped_visit`) and the `#decorate_visit` method. 
I'm quite happy extract some of those methods into their own classes or module, since they are closely related?

Reply:
👍